### PR TITLE
Make doc changes and small key ID fix

### LINKED ISSUE #137: Documentation: Enhance linux instructions
**Description**

I hit an error earlier: 
```
error: connecting to pscs: the Smart card resource manager is not running
```

We should add some docs for keyholders who hit this error to start and enable the pcsd daemon if they hit this issue
```
$ systemctl start pcscd.service
$ systemctl enable pcscd.service
```

### LINKED ISSUE #142: Check TUF Key ID output in verification
**Description**

Verifying keys now prints out the TUF key ID for better matching. However, this truncates the ID (probably an int conversion issue)

```
+ ./verify keys --root piv-attestation-ca.pem --key-directory /home/asraa/git/test-sigstore-root/ceremony/2022-03-09/keys

Outputting key verification and OpenSSL commands...

VERIFIED KEY 18361710
	TUF key id: 824636473168
```

### PR DESCRIPTION
**Pull request by asraa**
#### Ticket Link

Fixes https://github.com/sigstore/root-signing/issues/142
Fixes https://github.com/sigstore/root-signing/issues/137

#### Release Note

```release-note

```

### CODE REVIEW

File: KEYHOLDER.md [hunk 1]
```diff
@@ -30,6 +35,17 @@ You will be prompted to reset your hardware key and set a PIN. Choose a PIN betw
 
 This will output three files (a public key, device certificate, and hardware certificate) in a directory named with your serial number `ceremony/YYYY-MM-DD/keys/${SERIAL_NUM}`.
 
+Troubleshooting: If you hit the error
+```
+error: connecting to pscs: the Smart card resource manager is not running
+```
+
+then run the following to start the pcsc daemon:
```
Comment (asraa) [inline KEYHOLDER.md]: @haydentherapper did you have to install anything before running this step?

Comment (Hayden-IO) [reply KEYHOLDER.md]: Not on mac. On linux, `libpcsclite-dev`.

Comment (asraa) [reply KEYHOLDER.md]: Awesome, thanks! That's already included

**Review by dlorenc — DISMISSED** [review]

File: KEYHOLDER.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # TUF Generation
 
+Pre-requisites:
+* A local Git installation and a Go development setup
+* SSH authentication for GitHub (see [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh))
+* Keyholders: A USB port connection for your hardware key (beware of using a remote connection)
```
Comment (Hayden-IO) [inline KEYHOLDER.md]: "the keyholder should not assume that magic occurs during an SSH session"

File: KEYHOLDER.md [hunk 3]
```diff
@@ -30,6 +35,17 @@ You will be prompted to reset your hardware key and set a PIN. Choose a PIN betw
...
 
+Troubleshooting: If you hit the error
+```
+error: connecting to pscs: the Smart card resource manager is not running
+```
+
+then run the following to start the pcsc daemon:
+```
+systemctl start pcscd.service
+systemctl enable pcscd.service
```
Comment (Hayden-IO) [inline KEYHOLDER.md:46]: When I ran enable, it kept printing out information until I pressed ctrl-c. Have you seen this too?

**Review by Hayden-IO — DISMISSED** [review]

File: KEYHOLDER.md (hunk 3 above)
Comment (asraa) [reply KEYHOLDER.md:46]: hmmmm no! i'm surprised that it outputs logs, but that does remind me to warn that this will require root.

**Pull request merged**